hw/formal/tools/jaspergold/fpv.tcl

```diff
@@ -67,13 +67,15 @@ if {$stopat ne ""} {
 }
 
 if {[info exists ::env(AFTER_LOAD)]} {
-    if {$env(AFTER_LOAD) != ""} {
-        puts "Running prefix TCL command from $env(AFTER_LOAD)"
-        source $env(AFTER_LOAD)
+    set flist $env(AFTER_LOAD)
+    foreach file $flist {
+        if {$file != ""} {
```

Comment:
I don't *think* that it's possible for this to fail in TCL? I think that a list from an environment variable just ends up being a list of nonempty strings.

Reply:
I think the two tidy-ups you've made to this file have ended up in the wrong commit.

Trivial way to fix this:
- Use `git rebase -i` to edit the first commit.
- Then `git checkout SECOND_COMMIT hw/formal/tools/jaspergold/fpv.tcl`
- Then `git commit --amend`.
- Finally `git rebase --continue`.
- 
- And I think the change will have moved to the right place :-)